ui/components/ui/menu/menu-item.js

```diff
@@ -66,7 +66,7 @@ const MenuItem = React.forwardRef(
         />
       )}
       <div>
-        <div>{children}</div>
+        <Text as="div">{children}</Text>
```

Comment:
Updating the wrapping `div` to a `Text` component while preserving the HTML element as `div`.